src/functionalTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/BaseFunctionalTest.java

```diff
@@ -55,8 +56,23 @@ public void setUp() throws Exception {
         // Apply proxy for functional tests for all environments except preview
         // as due to NSG config it has to go through outbound proxy
         if (isProxyEnabled) {
-            Proxy proxy = new Proxy(Proxy.Type.HTTP, new InetSocketAddress(proxyHost, proxyPort));
+            Proxy proxy = new Proxy(
+                Proxy.Type.HTTP,
+                new InetSocketAddress(proxyHost, Integer.parseInt(proxyPort))
+            );
             OperationContext.setDefaultProxy(proxy);
+
+            // This is set temporary and will be removed/modified in subsequent PRs
+            System.setProperty("http.proxyHost", proxyHost);
+            System.setProperty("http.proxyPort", proxyPort);
+
+            System.setProperty("https.proxyHost", proxyHost);
```

Comment:
do we need different configs, variables for https?

Reply:
we want to try setting both so if not required going to remove in next PR

Reply:
variables can be same